pelican/__init__.py

```diff
@@ -327,6 +327,10 @@ def parse_arguments():
                         help=('Exit the program with non-zero status if any '
                               'errors/warnings encountered.'))
 
+    parser.add_argument('--dump-settings', dest='dump_settings',
+                        action='append', nargs='?', help='Read'
+                        'config file, dump final settings and exit.')
```

Comment:
1. There is no space between `Read` and `config`.
2. The help text does not describe what the optional parameter does.